### DISCUSSION TITLE
I can't renew my student subscription because there is no student mail

### DISCUSSION THREAD

**Discussion by tekerau666** [community/community#60679] [GitHub Education] [answered]
### Hi everyone,

**I am trying to renew my student subscription and I am asked to confirm my university's email, but my university does not provide us with personal emails to use. At the moment I have a personal account of the university's educational website and a valid student ID. What should I do to solve the problem?**

![Screenshot_1](https://github.com/community/community/assets/109969237/b68728ab-965d-4ee8-87bd-7fbbaecd6865)

**Comment by Christian-Schwarz2003** [comment]
Same here, I contacted support in April and got it removed for my school but now it's back and support hasn't responded in 2 weeks

**Comment by BenjaminDecaillet** [comment]
Hello,
similar issue here. I have a school issued e-mail for students but it is not matching what they request and I cannot re-submit my application.

I cannot reach out to support it seems now about global campus.

![Re-submit_Github](https://github.com/community/community/assets/26813570/bc1d581b-c09c-4b03-a2ab-6cbf9c6357a2)

**Comment by PavelS05** [comment]
Same here

My school doesn't provite it's students with email adresses that we can use to sign in in Global Campus

**Comment by ifeoluwaayo** [comment]
I also have the same issue. My school OOU doesn't have a student email for us, yet they're requesting one... Please they should look into it ASAP.

**Comment by nonvenomous** [comment]
Same issue here. I don't get why github simply assumes schools provide students with a school email.

**Comment by Marblog** [comment]
Some schools simply do not provide students with email addresses

**Reply by tekerau666** [reply]
Yes, but where I can submit this for github education team?

**Comment by tcitr-chen** [comment]
Me too!

**Comment by samus-aran** [comment, marked as answer]
Hi all 👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍
